project-addons/custom_emails_company/data/email_Payment_Order_advise_partners.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <!--Email template -->
+        <record id="email_template_edi_payment_order" model="mail.template">
```

Comment:
¿Cómo envías esta plantilla? no encuentro la lógica.
Actualmente se mandaría una similar con un módulo que ya tenéis instalado en preproducción, no se si será mejor que pises la plantilla que define ese módulo con la tuya, porque así no hay que hacer ninguna lógica de envio, el módulo en cuestión es: https://github.com/Comunitea/external_modules/tree/11.0/account_banking_sepa_mail

Reply:
Hola Omar. He pisado la plantilla del modulo "account_banking_sepa_mail" con la nuestra.